StringCalculatorKata/StringCalculatorKata/StringCalculator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+
+namespace StringCalculatorKata
+{
+    public class StringCalculator
+    {
+        private string _numbers;
+        private List<char> _delimiters = new List<char>() { ',', '\n' };
+
+        public int add(string numbers)
+        {
+            _numbers = numbers;
+            _checkForCustumDelimiter();
+            if (_numbers == "")
+                return 0;
+            return _numbers.Split(_delimiters.ToArray()).Select((num) => int.Parse(num)).Sum();
+        }
+
+        private void _checkForCustumDelimiter()
+        {
+            bool isDelimmeterSpecified = _numbers.StartsWith("//");
+            if (isDelimmeterSpecified == true)
+            {
+                char customDelimiter = _numbers.Substring(2, 1).ToCharArray()[0];
+                _delimiters.Add(customDelimiter);
```

Comment:
You shouldn't add these custom delimiters to the `_delimiters` list. This list should have the default (always allowed) delimiters. The custom delimiters you find here are only specific to the input you calculate.

Reply:
`StringCalculator` should be more abstract and not store data really.